src/useFind.ts

```diff
@@ -87,21 +88,32 @@ export type UseFindFunc<CustomApplication> = <
   params?: Ref<Params | undefined | null>,
 ) => UseFind<M>;
 
+type Options = {
+  disableUnloadingEventHandlers: boolean;
+  chunking: boolean;
```

Comment:
```suggestion
  useAllPages: boolean;
```

Should we maybe name it `useAllPages` or `loadAllPages`?

Reply:
@anbraten I used loadAllPages